test/client.test.js

```diff
@@ -4,13 +4,15 @@ import { Strategy } from '../lib/strategy';
 
 import CustomStrategy from './true_custom_strategy';
 import CustomFalseStrategy from './false_custom_strategy';
+import { UnleashEvents } from '../lib/events';
 
-function buildToggle(name, active, strategies, variants = []) {
+function buildToggle(name, active, strategies, variants = [], impressionData= false) {
```

Comment:
```suggestion
function buildToggle(name, active, strategies, variants = [], impressionData = false) {
```

I'm assuming this is the correct formatting.